src/main/java/org/opensearch/knn/index/query/ExactSearcher.java

```diff
@@ -98,10 +135,20 @@ private Map<Integer, Float> searchTopK(KNNIterator iterator, int k) throws IOExc
             final ScoreDoc doc = queue.pop();
             docToScore.put(doc.doc, doc.score);
         }
-
         return docToScore;
     }
 
+    private Map<Integer, Float> searchTopK(KNNIterator iterator, int k) throws IOException {
+        return searchTopCandidates(iterator, k, null);
```

Comment:
consider passing in `score -> true` to avoid nulls

nit: probably a wrapper method is not needed here since its 1 line of code

Reply:
Ack